packages/ui/src/CharacterTransition/index.stories.tsx

```diff
@@ -1,5 +1,10 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import { CharacterTransition } from '.';
+import styled from 'styled-components';
+
+const WhiteTextWrapper = styled.div`
+  color: ${props => props.theme.color.text.primary};
+`;
```

Comment:
`WhiteTextWrapper` _is_ required for this story, since `<CharacterTransition />` is unstyled text, and I didn't want to wrap this story in a `<Text />` or else consumers might think that it was styled out of the box.